### ISSUE TITLE
Remove delay for Collision Layer/Mask tooltip

### ISSUE THREAD

**Issue by Cerno-b** [godotengine/godot#84789]
### Godot version

v4.2.beta.custom_build [c7633951a]

### System information

Windows 10

### Issue description

Less of a bug, more of an annoyance:

The workflow for named Collision layers is to hover the mouse over the layer number to make the name appear. To make this actually work in practice, the tooltip delay should be removed for this widget. 

In order to find a specific collision layer by name, the dev would have to go over different candidates in quick succession to find the one they want. Waiting a second after each hover makes this feature rather hard to use

### Steps to reproduce

Inspector > CollisionObject2D > Collision

Hover over a collision layer or mask widget.

### Minimal reproduction project

Probably not required since very straightforward. I can provide one if necessary though.

**Comment by Cerno-b** [comment]
I would be willing to do this myself, I just need a hint on how, or whether this is even uniquely possible, since I am super new to the codebase.

**Comment by Mickeon** [comment]
Because of the way tooltips are organised this is currently not feasible. Something like https://github.com/godotengine/godot/pull/64796 would make this possible, however.

**Comment by Cerno-b** [comment]
@Mickeon Thanks for the info. So would this be a duplicate (to be closed) or a follow-up (to be left open)?

**Comment by Cerno-b** [comment]
Nevermind, the link is a PR, so it would be a follow-up. I'll leave it open and wait for the merge.

**Comment by Calinou** [comment]
This may be better done as a custom display rather than removing tooltip delay for a specific control. Tooltips spawn at unpredictable locations, while custom display can be made to show always at the same location (just above/below the list of icons).

**Comment by Cerno-b** [comment]
@Calinou Works for me. I don't know much about the widgets Godot uses, but maybe I can give this a try. 

Any hints to make my life easier? I suppose it should be one display field for both the layer and the mask to conserve space and then disable the tooltip for the existing widgets because the info will be shown in the permanent panel instead?

**Comment by Calinou** [comment]
> Any hints to make my life easier? I suppose it should be one display field for both the layer and the mask to conserve space and then disable the tooltip for the existing widgets because the info will be shown in the permanent panel instead

Yes, the existing tooltips should be removed if we go the custom Label route.

**Comment by timothyqiu** [comment]
> The workflow for named Collision layers is to hover the mouse over the layer number to make the name appear.

You can use the three-dots menu for that. No need to use the tooltip.

![Peek 2024-04-20 18-00](https://github.com/godotengine/godot/assets/372476/8c0a8ec0-edfc-4214-a40e-2cb93d29112f)

**Comment by romlok** [comment]
I'd like to throw out another suggestion that I think would make the layer/mask UI easier to understand and use at a glance:

- Replace the number on a button with the first character of its custom layer name.

There will often likely be duplicates, but the combination of the character and visual position is a better guide than just the position IME.
Alternatively, the first two characters, or first two initials (eg. "PH" for the "Player Hurtbox" layer) could be shown, but the boxes may have to be widened (or the font reduced) to do that.
